llvm/lib/Target/RISCV/MCTargetDesc/RISCVTargetStreamer.cpp

```diff
@@ -34,7 +34,7 @@ void RISCVTargetStreamer::emitDirectiveOptionNoRVC() {}
 void RISCVTargetStreamer::emitDirectiveOptionRelax() {}
 void RISCVTargetStreamer::emitDirectiveOptionNoRelax() {}
 void RISCVTargetStreamer::emitDirectiveOptionArch(
-    ArrayRef<RISCVOptionArchArg> Args) {}
+    const ArrayRef<RISCVOptionArchArg> &Args) {}
```

Comment:
ArrayRef should always be passed by value. It's cheap to copy. It's a pointer and size.

Reply:
Good to know - thanks